CardForm/src/main/java/com/braintreepayments/cardform/view/CardEditText.java

```diff
@@ -98,6 +129,16 @@ public void afterTextChanged(Editable editable) {
 
             if (isValid()) {
                 focusNextView();
+
+                if (mMasked) {
+                    maskNumber();
```

Comment:
Isn't this already done in `onFocusChanged`? Is there any conflict/race condition including this here?

Reply:
I don't think there are any race conditions, the issue was that the focus change listener was not called in our unit tests after a call to `type()` I believe. This can be fixed by calling the focus change listener manually in the test like the other tests.